macros/stitch/base/stitch_fb_ad_insights.sql

```diff
@@ -28,6 +27,34 @@ select
     inline_post_engagement,
     unique_inline_link_clicks
 
+from
+    {{ var('ads_insights_table') }}
+
+{% endmacro %}
+
+
+{% macro snowflake__stitch_fb_ad_insights() %}
```

Comment:
these two versions are nearly identical. is there a reason we need an extra verison?

Reply:
@jthandy There were many nullifs() being used that were throwing errors for numeric fields in snowflake, instead of deleting the nullifs() in the default adapter, I created a snowflake adapter to remove the nullifs that I could.

Reply:
@jthandy these were removed, ready for another review